website/src/components/CLIDownloadScript.tsx

```diff
@@ -1,15 +0,0 @@
-import { usePluginData } from "@docusaurus/useGlobalData";
-import CodeBlock from "@theme/CodeBlock";
-import React from "react";
-
-export default function CLIDownloadScript(): JSX.Element {
-  const latestVersion = usePluginData("latestCLIReleaseTag") as { tag: string };
```

Comment:
I think there's more code around this `latestCLIReleaseTag` that may be unused now?

Reply:
Basically it seems like you want to revert #1069 and then modify cli.md.

Reply:
Oh, I see. Makes sense 👍